lib/api/expect/_baseExpect.js

```diff
@@ -147,7 +147,7 @@ class ExpectInstance extends EventEmitter {
   }
 
   retryCommand() {
-    const promise = this.command();
+    const promise = this.command(...this.commandArgs);
```

Comment:
this isn't necessary, you can just pass the `this.commandArgs` inside the command.

Reply:
so if we retry with arguments, then in future retry it will send `[[<args>]]`, which will break things, as spread syntax allows array elements to be passed in as individual arguments, but here the individual argument is `[<args>]`